CONTRIBUTING.md

```diff
@@ -63,7 +63,7 @@ bundle exec bin/github-linguist --breakdown
 ## Adding an extension to a language
 
 We try only to add new extensions once they have some usage on GitHub.
-In most cases we prefer that extensions be in use in hundreds of repositories before supporting them in Linguist.
+In most cases we prefer that extensions be in use in at least 200 unique user/repo repositories before supporting them in Linguist.
```

Comment:
The statement "user/repo" can be a bit confusing at first glance if you take the slash to mean "or" instead of a URI separator. What would be a non-"user/repo" repository?

Reply:
You're looking at an old commit 😉. I thought that moments after I pushed, hence I changed it in the next commit.

Reply:
Ah, didn't see that, GitHub seems to like deleting the bottom half of the page some of the time...